Add an email announcing the SR2025 competition (#188)
srobo/team-emails: SR2025/2024-09-02-competition-announcement.md

---
to: Potential team supervisors
subject: Announcing Student Robotics 2025!
---

Hi,

We're pleased to [announce][announcement] that registration for Student Robotics
2025 is now open. If you would like to compete, please complete the
[signup form][signup-form] with the details of your team. All future emails
about SR2025 will be sent to the details you provide there.

This year we're hosting [Kickstart][kickstart] at the University of Southampton,
on Saturday 12th October. We will also be live-streaming the event for those
unable to attend in person. We expect to confirm places towards the end of

> September.

Author: Slightly concerned that "end of September" -> 12th October feels quite short -- do we think that this might put people off? Especially since we're only announcing at the start of September.

Author: https://github.com/srobo/team-emails/pull/188/commits/bede9ebdebe9339da14e64581ee2a055f83d07b7

As usual we will loan your team a [kit][kit] of parts to help build your robot.
This year we hope to send you your kit before Kickstart. We will also be sharing
some [pre-Kickstart activities][pre-kickstart-activities] to enable your team to

> start learning about the kit.

Author: TODO:
- the activities are about robotics and the simulator, not the kit
- we're publishing them at the same time as this email goes out, so the tenses here should be updated

Author: e443d9f

---
to: Potential team supervisors
subject: Announcing Student Robotics 2025!
---

Hi,

We're pleased to [announce][announcement] that registration for Student Robotics
2025 is now open. If you would like to compete, please complete the
[signup form][signup-form] with the details of your team. All future emails
about SR2025 will be sent to the details you provide there.

This year we're hosting [Kickstart][kickstart] at the University of Southampton,
on Saturday 12th October. We will also be live-streaming the event for those
unable to attend in person. We expect to confirm places towards the latter half
of September. Places are limited, so sign up soon to avoid disappointment.

As usual we will loan competing teams a [kit][kit] of parts to help build their
robots. This year we hope to send out kits before Kickstart.

> We have also prepared some [pre-Kickstart activities][pre-kickstart-activities]

Reviewer: We could highlight the pre kickstart activities with some emphasis on the fact they are new so they are even less likely to be missed?
```suggestion
Also **new for this year** we have prepared some [pre-Kickstart activities][pre-kickstart-activities]
```

> We're pleased to [announce][announcement] that registration for Student Robotics
> 2025 is now open. If you would like to compete, please complete the
> [signup form][signup-form] with the details of your team. All future emails
> about SR2025 will be sent to the details you provide there.
> 
> This year we're hosting [Kickstart][kickstart] at the University of Southampton,
> on Saturday 12th October. We will also be live-streaming the event for those
> unable to attend in person. We expect to confirm places towards the latter half
> of September. Places are limited, so sign up soon to avoid disappointment.
> 
> As usual we will loan competing teams a [kit][kit] of parts to help build their
> robots. This year we hope to send out kits before Kickstart.
> 
> We have also prepared some [pre-Kickstart activities][pre-kickstart-activities]
> to enable teams to start learning about robotics and our simulator right away.
> 
>   [I'm interested in bringing a team to SR2025][signup-form]

Reviewer 2: I think breaking it into sections makes it easier to convey all the points we are trying to get across here.

```suggestion
We're pleased to [announce][announcement] that registration for Student Robotics
2025 is now open.

## Signups
If you would like to compete, please complete the
[signup form][signup-form] with the details of your team. All future emails
about SR2025 will be sent to the details you provide there.

  [I'm interested in bringing a team to SR2025][signup-form]

## Kickstart

This year we're hosting [Kickstart][kickstart] at the University of Southampton,
on Saturday 12th October. We will also be live-streaming the event for those
unable to attend in person. We expect to confirm places towards the latter half
of September. Places are limited, so sign up soon to avoid disappointment.

As usual we will loan competing teams a [kit][kit] of parts to help build their
robots. This year we hope to send out kits before Kickstart.

## Pre-Kickstart Activities

The [pre-Kickstart activities][pre-kickstart-activities] are a new concept we are introducing with SR2025. They use our simulator for a set of activities intended to be completed between signups and Kickstart. 
These will teach some core concepts of robotics and introduce some simple sensors that teams could use on their physical robots.

```

Author: Thanks for the suggestion. I can see what you're going for, though in this case I'm not sure that the suggested change is ideal. Among other things it moves the CTA to the top, which greatly reduces the chance that people will read the rest of the email (having it at the bottom was deliberate to counter this).

 The email is also pretty short, so I expect people will be able to read it all without needing it breaking up in to sections. I'll revisit this once we've got the links to the other general information in place and see if that changes the balance.

---

Hi,

We're pleased to [announce][announcement] that registration for Student Robotics
2025 is now open. The [programme][programme-structure] will run from October
this year until around Easter 2025.

If you would like to compete, please complete the [signup form][signup-form]
with the details of your team. Entries must be made by a responsible adult on
behalf of a team -- we call this role a [team supervisor][team-supervisor]. All
future emails about SR2025 will be sent to the details you provide via the form.

This year we're hosting [Kickstart][kickstart] at the University of Southampton,
on Saturday 12th October. We will also be live-streaming the event for those
unable to attend in person. We expect to confirm places towards the latter half
of September. Places are limited, so sign up soon to avoid disappointment.

As usual we will loan competing teams a [kit][kit] of parts to help build their
robots. This year we hope to send out kits before Kickstart.

Also **new for this year** we have prepared some [pre-Kickstart activities][pre-kickstart-activities]
to enable teams to start learning about robotics and our simulator right away.

>   [I'm interested in bringing a team to SR2025][signup-form]

Author: ```suggestion
  [Sign up to SR2025][signup-form]
```
